go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	github.com/gdamore/tcell/v2 v2.2.0
 	github.com/gizak/termui/v3 v3.1.0
 	github.com/gogo/googleapis v1.4.0 // indirect
-	github.com/golang/protobuf v1.5.1
+	github.com/golang/protobuf v1.5.2
```

Comment:
The whole module is deprecated now. Does it pass `make lint`?

Reply:
yep, I think there's some legacy we need eventually get rid of to remove this one